### DISCUSSION TITLE
i am using google auth but code always error  althoug it at first can true why ?

### DISCUSSION THREAD

**Discussion by hamdyadam97** [community/community#137548] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

i am using google auth but code always error  althoug it at first can true why ?

**Comment by Azymack** [comment]
pls be specific with the issue.
Sharing your current code will be helpful.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @hamdyadam97 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
